Projects/FeatureEncyclopedia/Scene/Thumbnail/ThumbnailView.swift

```diff
@@ -26,34 +26,36 @@ public struct ThumbnailView: View {
             .frame(width: 110, height: 210)
             .overlay {
                 VStack {
-                    Rectangle()
-                        .frame(width: 56, height: 114)
-                    Text(makId)
+                    MakHolyImageView(imageId: mak.makImg!, type: .middle)
```

Comment:
```suggestion
                    MakHolyImageView(imageId: mak.makImg ?? -1, type: .middle)
```
p3
makImg가 안들어오지는 않을 거 같은데.. 혹시나 모르니까 이런 방법으로 옵셔널을 처리하는게 어떨까요?

Reply:
-1은 뭐에요??

Reply:
-1이 의미가 있다기 보다는, 그냥 예시를 들었습니다. 만약 makImg가 null로 들어오면 앱이 죽어버리기 때문에 에러 로직 처리를 하면 좋겠다. 라는 뜻이에요!